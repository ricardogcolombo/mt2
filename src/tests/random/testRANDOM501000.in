50 1000
1 15 111 5 99
1 31 81 3 127
1 29 28 18 3
1 19 128 46 34
1 43 87 19 109
1 8 32 31 22
1 45 5 44 61
1 5 24 32 77
1 11 70 8 14
1 17 54 46 81
1 39 101 4 55
1 39 84 12 108
1 11 88 49 52
1 8 27 10 90
1 3 104 39 50
1 34 71 21 63
1 22 79 19 89
1 30 93 1 56
1 1 42 15 69
1 12 50 14 51
1 49 22 2 17
1 5 67 44 31
1 23 104 50 0
1 40 27 32 51
1 33 29 17 33
1 21 120 50 58
1 1 92 18 11
1 21 63 29 64
1 12 100 42 81
1 7 8 12 112
1 36 29 6 116
1 15 103 43 91
1 25 117 6 107
1 29 82 33 23
1 1 103 38 106
1 30 83 27 9
1 38 129 43 81
1 4 1 35 101
1 34 24 44 73
1 43 95 27 56
1 4 34 12 97
1 13 100 45 2
1 5 69 9 41
1 1 91 25 98
1 3 26 13 70
1 48 27 29 92
1 4 45 9 38
1 33 18 6 86
1 18 35 5 21
1 2 121 22 113
1 11 28 10 12
1 41 93 24 130
1 47 6 35 73
1 20 52 38 11
1 40 1 24 50
1 3 112 44 72
1 9 91 38 67
1 2 88 6 50
1 36 61 42 81
1 26 18 2 30
1 29 109 49 36
1 7 3 8 73
1 47 4 50 55
1 5 83 41 36
1 9 11 10 55
1 30 125 29 39
1 20 73 41 59
1 41 126 39 34
1 48 70 31 40
1 5 60 38 9
1 46 43 8 27
1 17 13 44 85
1 46 35 36 105
1 24 62 44 68
1 35 91 44 68
1 27 51 16 5
1 29 23 17 65
1 14 107 16 116
1 6 129 39 126
1 25 117 2 101
1 24 118 4 113
1 29 84 21 20
1 39 92 50 121
1 16 92 23 63
1 49 60 1 16
1 37 5 31 35
1 27 30 23 17
1 22 2 33 43
1 22 92 27 115
1 15 8 5 27
1 32 38 21 34
1 15 36 39 27
1 13 2 40 43
1 7 14 14 7
1 44 116 14 72
1 41 62 13 6
1 42 20 49 91
1 13 17 9 54
1 23 110 15 84
1 36 70 43 88
1 34 43 13 75
1 32 13 47 96
1 45 44 46 116
1 28 49 18 113
1 37 130 39 21
1 47 87 39 65
1 27 91 18 32
1 36 120 25 112
1 40 80 3 50
1 23 11 7 50
1 11 11 29 60
1 29 80 19 40
1 44 29 48 103
1 22 15 17 21
1 33 96 19 44
1 43 47 16 75
1 20 50 46 92
1 19 25 10 4
1 36 107 6 32
1 12 23 2 119
1 28 10 21 114
1 32 81 48 6
1 49 95 11 24
1 3 78 43 40
1 4 31 39 88
1 25 10 33 93
1 32 67 19 53
1 6 104 49 5
1 30 52 19 82
1 1 22 23 25
1 27 45 37 60
1 38 57 4 81
1 40 11 11 54
1 20 36 33 12
1 49 119 44 10
1 13 109 10 97
1 30 112 36 124
1 35 129 5 19
1 2 62 49 54
1 16 111 24 12
1 16 98 17 62
1 48 67 11 95
1 19 8 15 98
1 31 17 17 127
1 5 41 15 27
1 6 87 30 3
1 40 16 11 130
1 24 86 21 95
1 46 70 41 112
1 4 62 26 94
1 36 129 5 84
1 7 98 4 25
1 36 15 33 70
1 32 106 10 43
1 42 124 11 123
1 43 38 14 102
1 26 33 27 90
1 22 5 1 127
1 10 28 38 107
1 7 110 50 108
1 39 62 25 80
1 37 31 21 130
1 20 126 6 115
1 34 57 17 91
1 45 121 48 46
1 26 107 34 18
1 29 77 45 106
1 48 22 36 29
1 6 113 45 51
1 47 0 25 10
1 35 36 14 68
1 30 52 37 29
1 14 80 19 126
1 28 113 10 25
1 26 40 18 41
1 32 71 30 63
1 40 91 29 128
1 29 118 45 121
1 37 0 33 64
1 50 80 22 48
1 26 108 50 83
1 1 61 45 83
1 44 40 47 17
1 2 103 35 21
1 27 28 49 66
1 32 71 10 65
1 47 75 15 129
1 46 121 43 82
1 15 27 11 36
1 17 43 10 3
1 19 46 17 34
1 18 38 5 91
1 16 77 14 62
1 11 28 15 54
1 46 117 1 101
1 16 88 7 86
1 10 26 11 45
1 49 125 12 4
1 21 61 45 22
1 33 23 7 45
1 3 56 42 67
1 5 126 29 68
1 13 69 15 76
1 37 64 21 92
1 34 69 6 5
1 47 61 4 84
1 9 22 45 119
1 14 98 43 7
1 19 5 26 93
1 13 51 34 7
1 29 21 20 104
1 14 106 47 35
1 40 98 18 36
1 37 19 3 36
1 22 56 15 63
1 27 21 50 109
1 2 49 4 105
1 22 39 9 122
1 12 0 23 15
1 50 88 10 55
1 5 58 18 117
1 11 25 38 80
1 5 88 15 8
1 35 16 18 70
1 10 128 41 84
1 12 108 47 52
1 13 12 45 120
1 47 72 23 55
1 5 48 48 86
1 39 30 48 84
1 40 120 28 128
1 4 35 8 116
1 17 69 11 94
1 46 97 39 22
1 7 58 22 109
1 50 30 40 111
1 31 126 25 39
1 24 103 26 102
1 19 127 38 69
1 11 47 13 29
1 48 91 6 4
1 25 21 28 53
1 35 81 29 123
1 25 7 44 125
1 32 114 33 45
1 44 83 5 27
1 14 92 12 15
1 2 60 45 101
1 10 109 6 52
1 35 48 40 102
1 41 28 16 72
1 7 59 29 38
1 44 2 48 129
1 19 10 14 101
1 45 68 19 75
1 5 23 27 85
1 32 23 9 72
1 16 6 11 3
1 22 95 4 86
1 25 87 39 36
1 13 35 33 109
1 40 118 19 72
1 9 53 14 84
1 12 15 14 35
1 5 91 4 30
1 8 70 35 49
1 10 36 6 18
1 4 123 20 15
1 7 0 17 79
1 7 100 14 52
1 28 98 50 118
1 27 105 2 61
1 44 18 42 35
1 12 31 3 89
1 13 102 17 74
1 17 5 19 54
1 23 108 31 84
1 39 97 29 19
1 25 110 33 15
1 4 16 24 28
1 24 58 4 2
1 43 81 29 98
1 15 125 38 53
1 41 98 24 36
1 13 113 50 104
1 9 61 29 128
1 38 4 22 130
1 18 105 8 101
1 28 44 14 59
1 18 18 40 121
1 9 36 1 30
1 48 7 40 112
1 20 15 35 92
1 10 68 35 128
1 8 26 16 17
1 40 85 49 125
1 19 59 42 0
1 14 39 16 90
1 14 17 35 8
1 31 49 25 77
1 2 14 46 45
1 8 93 22 18
1 29 40 20 104
1 28 77 12 39
1 25 46 18 58
1 31 75 28 43
1 12 37 19 24
1 5 103 30 18
1 34 100 32 101
1 23 123 30 122
1 22 90 43 62
1 33 129 27 30
1 10 111 48 17
1 40 121 37 112
1 32 30 21 39
1 8 113 6 85
1 27 15 32 78
1 7 68 30 35
1 35 35 42 71
1 8 123 42 22
1 50 63 2 24
1 33 96 31 115
1 11 41 25 24
1 48 93 39 127
1 8 24 21 36
1 6 0 12 87
1 30 56 42 93
1 32 89 6 67
1 39 19 44 108
1 44 53 20 65
1 26 81 46 1
1 43 40 47 45
1 47 25 8 120
1 50 128 18 0
1 13 16 21 50
1 41 1 3 122
1 29 119 36 96
1 12 98 42 72
1 12 103 40 59
1 22 109 18 66
1 29 88 20 25
1 16 46 27 111
1 44 47 16 111
1 39 9 34 106
1 47 38 19 127
1 13 123 46 27
1 34 113 49 65
1 16 17 18 118
1 36 78 5 56
1 24 96 12 94
1 26 6 1 32
1 27 33 9 4
1 41 16 12 51
1 21 4 4 51
1 42 75 26 9
1 50 83 9 32
1 38 110 9 44
1 26 121 47 36
1 10 66 6 14
1 22 68 17 72
1 6 117 35 116
1 50 2 1 45
1 13 40 40 96
1 2 101 46 93
1 22 94 13 119
1 9 37 8 21
1 27 29 18 110
1 27 106 45 93
1 27 21 38 20
1 3 47 26 60
1 18 130 21 74
1 16 110 6 83
1 29 20 36 114
1 42 100 21 50
1 24 66 36 11
1 11 68 43 86
1 38 123 15 46
1 38 127 45 63
1 6 54 39 66
1 33 130 34 118
1 6 102 38 94
1 22 85 49 26
1 46 8 47 129
1 12 78 30 88
1 5 86 43 78
1 16 7 24 52
1 39 49 32 51
1 8 58 23 4
1 42 56 6 89
1 41 57 19 86
1 13 116 34 37
1 8 82 5 122
1 44 1 10 115
1 29 32 45 7
1 41 29 18 123
1 14 2 23 85
1 21 129 32 99
1 40 129 21 118
1 14 105 36 124
1 8 39 35 52
1 13 103 38 49
1 36 48 44 101
1 22 59 44 93
1 37 75 26 7
1 2 120 48 26
1 15 26 37 23
1 35 35 40 38
1 47 68 25 76
1 5 93 15 106
1 43 78 1 69
1 13 86 3 117
1 40 62 28 111
1 4 106 38 0
1 40 114 18 101
1 34 128 15 48
1 6 78 27 67
1 36 64 35 85
1 4 16 8 11
1 31 34 42 18
1 48 47 18 121
1 31 20 39 102
1 27 41 15 76
1 31 104 14 115
1 50 23 39 91
1 4 24 34 94
1 50 96 34 93
1 49 97 17 24
1 38 84 21 113
1 48 92 39 90
1 41 105 8 128
1 41 59 10 85
1 17 103 46 126
1 45 28 4 112
1 28 88 9 52
1 8 120 20 4
1 35 95 44 53
1 23 95 50 58
1 6 130 34 61
1 27 22 1 89
1 3 94 15 106
1 26 79 1 118
1 2 127 49 103
1 6 62 23 36
1 50 117 19 68
1 26 91 48 63
1 42 104 35 124
1 9 99 4 35
1 46 23 22 121
1 6 128 42 56
1 7 50 32 27
1 11 83 7 125
1 28 10 19 125
1 11 98 20 90
1 35 0 4 49
1 41 102 30 16
1 22 33 10 129
1 18 35 6 81
1 40 43 19 7
1 17 6 15 55
1 18 58 35 14
1 8 40 12 41
1 31 108 12 37
1 30 22 17 79
1 43 19 6 29
1 18 50 23 42
1 39 68 43 123
1 14 121 15 65
1 8 23 40 21
1 18 9 16 42
1 2 115 1 42
1 29 82 1 70
1 18 31 20 99
1 19 121 30 37
1 30 43 31 124
1 11 99 47 86
1 35 38 14 6
1 40 70 13 61
1 22 35 3 78
1 36 120 31 33
1 29 115 15 40
1 3 60 36 45
1 42 116 36 54
1 49 86 4 52
1 48 37 29 110
1 20 12 10 7
1 47 48 32 129
1 32 70 16 11
1 35 45 43 110
1 36 22 29 26
1 17 62 13 81
1 32 35 15 2
1 49 60 8 121
1 21 15 10 37
1 22 15 14 96
1 22 22 24 70
1 9 46 46 64
1 50 19 13 28
1 34 122 21 117
1 20 4 8 49
1 7 52 19 89
1 8 13 17 94
1 19 44 47 32
1 40 61 45 127
1 21 24 3 29
1 8 115 3 71
1 40 7 7 91
1 37 96 22 101
1 4 56 28 129
1 7 66 4 53
1 23 83 43 19
1 16 104 46 63
1 3 29 21 10
1 36 110 7 67
1 34 60 2 83
1 1 50 8 1
1 4 15 3 38
1 35 61 36 125
1 2 93 33 63
1 29 115 18 28
1 8 101 28 98
1 39 89 1 107
1 38 70 36 4
1 10 14 18 71
1 16 71 4 11
1 43 69 6 41
1 12 115 43 15
1 41 59 27 54
1 8 32 26 74
1 36 73 44 77
1 42 18 27 5
1 23 72 37 39
1 35 64 19 85
1 30 112 49 101
1 4 30 42 68
1 11 25 37 18
1 50 17 34 20
1 18 80 4 86
1 3 61 30 66
1 25 124 42 113
1 2 64 36 124
1 20 126 41 30
1 5 33 31 114
1 11 60 22 86
1 7 2 49 17
1 44 72 12 38
1 16 55 27 117
1 31 40 21 50
1 41 69 35 124
1 38 107 21 16
1 33 98 7 102
1 8 92 19 66
1 35 70 20 0
1 14 89 21 118
1 30 10 50 38
1 3 74 33 17
1 15 18 22 80
1 43 87 49 82
1 2 56 7 38
1 16 106 12 33
1 9 33 43 95
1 40 87 7 76
1 46 39 5 128
1 7 52 26 128
1 7 19 46 18
1 8 27 33 89
1 8 98 23 108
1 28 104 49 7
1 41 34 50 24
1 40 117 5 112
1 17 67 2 94
1 2 27 33 128
1 18 33 11 125
1 16 3 21 84
1 10 45 27 3
1 4 55 23 130
1 50 91 34 76
1 9 116 45 95
1 33 129 48 7
1 31 69 18 116
1 44 18 37 48
1 1 64 50 128
1 43 40 47 120
1 41 6 44 67
1 25 49 2 111
1 46 117 19 50
1 20 42 43 88
1 13 117 21 87
1 43 71 12 33
1 23 20 15 114
1 1 63 35 14
1 4 38 35 42
1 42 93 32 63
1 40 125 36 108
1 24 114 43 123
1 50 66 18 99
1 24 11 34 4
1 39 26 21 35
1 2 27 37 109
1 34 97 49 112
1 11 102 36 5
1 16 92 11 79
1 34 30 21 6
1 25 51 24 63
1 3 3 47 41
1 1 40 47 99
1 40 19 15 122
1 17 103 2 30
1 7 81 34 114
1 14 85 49 6
1 1 124 9 22
1 26 47 21 45
1 25 118 48 66
1 24 37 12 52
1 36 12 32 105
1 2 43 28 53
1 11 41 30 118
1 32 114 23 68
1 24 5 14 22
1 22 32 17 104
1 23 9 49 85
1 36 92 47 42
1 13 36 28 46
1 9 47 17 46
1 18 96 49 127
1 37 93 6 19
1 23 89 2 55
1 11 81 41 29
1 10 130 33 85
1 26 52 7 10
1 20 39 1 20
1 8 49 6 29
1 32 8 46 78
1 28 59 13 4
1 12 44 20 89
1 13 23 50 48
1 44 111 50 52
1 20 51 14 109
1 23 81 11 129
1 17 84 33 53
1 16 103 23 35
1 39 126 18 130
1 45 10 19 76
1 35 91 45 87
1 32 99 4 105
1 5 30 11 92
1 20 46 29 90
1 26 95 16 50
1 33 10 11 112
1 39 125 35 59
1 27 58 3 105
1 18 103 47 101
1 35 15 45 103
1 48 32 40 56
1 28 41 22 14
1 37 108 44 22
1 47 60 33 47
1 31 63 36 119
1 20 68 34 82
1 5 26 12 115
1 26 21 42 99
1 2 69 25 116
1 13 119 12 66
1 5 9 48 30
1 36 121 31 3
1 5 13 50 50
1 15 104 46 103
1 21 74 46 124
1 33 21 4 63
1 11 123 23 61
1 30 54 2 70
1 20 94 35 32
1 49 25 26 55
1 27 54 41 102
1 3 21 7 99
1 24 79 16 26
1 41 76 1 118
1 45 101 50 100
1 25 77 5 112
1 9 114 47 117
1 38 79 35 110
1 37 117 12 51
1 35 56 49 44
1 45 106 41 49
1 28 61 18 126
1 49 26 50 63
1 21 91 43 65
1 25 76 3 55
1 45 8 6 29
1 17 87 39 82
1 28 25 34 66
1 5 109 7 113
1 44 102 34 58
1 38 7 33 69
1 16 1 28 11
1 20 96 9 35
1 1 125 3 52
1 22 27 35 78
1 6 78 25 41
1 17 105 29 122
1 45 31 11 17
1 26 102 23 59
1 2 5 40 98
1 30 7 24 12
1 36 70 24 72
1 39 91 31 105
1 49 84 41 100
1 2 62 35 118
1 41 128 25 57
1 23 62 35 50
1 20 19 1 64
1 26 59 43 5
1 30 49 46 9
1 15 72 7 128
1 27 114 30 123
1 25 17 23 48
1 28 4 3 35
1 6 14 43 69
1 34 41 2 128
1 49 41 24 127
1 12 89 5 46
1 28 25 18 59
1 38 123 17 54
1 37 114 16 91
1 26 101 32 47
1 21 91 40 16
1 44 103 7 50
1 34 86 7 35
1 41 11 2 111
1 21 30 44 83
1 37 2 18 72
1 28 67 41 4
1 34 34 13 108
1 2 82 14 127
1 40 0 8 62
1 1 24 15 20
1 49 70 23 59
1 18 21 31 41
1 18 94 29 115
1 2 40 37 27
1 47 11 31 86
1 5 40 45 124
1 41 31 45 58
1 37 85 38 98
1 23 13 42 47
1 4 91 8 67
1 37 99 45 27
1 3 8 33 98
1 17 23 2 129
1 20 56 48 10
1 22 61 45 101
1 2 45 8 112
1 10 55 12 14
1 11 56 9 29
1 41 24 46 66
1 30 123 37 2
1 37 72 13 71
1 28 56 11 114
1 43 54 26 95
1 41 130 14 14
1 46 28 17 54
1 2 23 22 92
1 1 38 16 24
1 45 65 13 44
1 18 44 38 27
1 38 4 30 92
1 10 107 32 70
1 18 109 11 42
1 5 80 33 11
1 35 2 5 100
1 19 11 20 77
1 20 74 6 57
1 4 7 14 48
1 26 8 13 119
1 27 105 37 103
1 33 95 2 71
1 31 19 41 49
1 42 89 47 108
1 50 84 13 36
1 44 70 34 55
1 48 42 13 109
1 18 5 17 46
1 49 49 10 78
1 46 125 18 15
1 14 26 6 75
1 2 36 32 22
1 15 112 32 55
1 36 2 5 123
1 47 93 39 94
1 16 73 14 98
1 8 38 39 57
1 45 4 50 57
1 26 54 22 37
1 33 6 31 46
1 27 27 24 93
1 15 15 30 59
1 35 30 2 1
1 26 75 49 30
1 34 122 42 40
1 23 66 49 88
1 21 115 41 130
1 15 86 41 124
1 10 98 15 101
1 4 84 25 110
1 34 13 17 129
1 17 34 20 118
1 29 107 19 0
1 37 45 7 41
1 50 105 17 38
1 46 65 20 60
1 3 58 23 65
1 41 122 7 5
1 33 77 26 128
1 24 84 8 117
1 19 67 15 110
1 47 65 29 111
1 41 31 47 91
1 6 48 44 128
1 12 30 41 10
1 23 56 22 71
1 48 15 25 117
1 42 16 6 88
1 27 70 26 47
1 26 89 19 12
1 12 24 14 11
1 20 100 39 63
1 37 88 50 53
1 21 5 19 73
1 35 94 25 100
1 49 105 2 124
1 41 123 11 124
1 39 90 12 104
1 16 55 49 68
1 39 1 45 74
1 34 97 19 90
1 32 102 15 97
1 14 74 29 105
1 45 76 7 13
1 21 87 31 72
1 24 28 7 29
1 48 106 42 18
1 5 68 30 4
1 10 19 33 110
1 33 0 21 50
1 27 105 31 46
1 16 41 44 122
1 40 100 39 77
1 13 19 18 66
1 18 84 14 43
1 35 39 37 3
1 25 129 29 2
1 22 40 1 45
1 9 53 14 40
1 31 32 50 126
1 13 16 10 31
1 9 91 27 35
1 19 116 43 53
1 2 20 37 10
1 39 69 26 61
1 38 0 43 93
1 34 54 28 100
1 4 19 41 121
1 48 86 28 81
1 42 7 49 96
1 12 41 11 58
1 31 89 38 70
1 47 108 19 2
1 3 4 16 46
1 40 118 15 89
1 7 41 17 69
1 4 31 10 44
1 16 73 34 66
1 6 69 45 23
1 16 37 47 38
1 39 100 12 40
1 39 49 13 70
1 20 53 24 16
1 34 52 30 100
1 12 25 25 89
1 30 122 43 78
1 44 4 17 17
1 1 48 35 83
1 7 13 28 23
1 37 36 30 15
1 47 46 9 123
1 27 82 33 67
1 14 104 47 126
1 36 67 4 118
1 41 94 30 81
1 41 4 38 121
1 27 55 35 1
1 6 62 20 109
1 45 9 21 5
1 25 5 19 49
1 49 24 24 36
1 15 51 45 49
1 24 53 32 76
1 20 119 29 43
1 1 88 47 109
1 23 46 25 125
1 6 83 44 27
1 43 23 5 1
1 40 125 13 70
1 15 39 12 24
1 7 43 25 99
1 12 70 25 59
1 41 27 24 67
1 23 79 25 33
1 39 54 20 39
1 10 75 7 25
1 31 85 32 68
1 4 75 15 66
1 10 90 26 63
1 34 3 29 58
1 27 46 13 48
1 39 47 15 107
1 13 86 23 72
1 46 98 24 51
1 25 30 49 116
1 6 115 12 101
1 22 102 8 5
1 27 59 23 57
1 40 28 36 92
1 29 125 15 62
1 46 31 32 113
1 40 83 30 112
1 33 26 36 86
1 18 89 36 106
1 7 42 6 106
1 15 87 45 97
1 46 84 34 64
1 25 8 35 55
1 22 74 25 24
1 18 73 4 41
1 49 106 3 12
1 17 79 4 61
1 5 113 19 90
1 12 79 27 29
1 12 55 24 53
1 18 33 44 96
1 42 113 21 57
1 41 27 26 89
1 24 57 18 5
1 13 112 21 67
1 19 116 6 41
1 39 46 6 129
1 11 69 31 6
1 36 97 3 92
1 14 16 13 8
1 48 43 28 87
1 44 97 34 90
1 33 9 34 33
1 4 5 16 57
1 22 13 34 81
1 1 126 29 101
1 39 108 27 84
1 20 114 43 84
1 19 65 27 29
1 25 34 22 51
1 47 82 35 90
1 3 127 7 99
1 16 29 42 103
1 31 26 43 93
1 48 75 28 65
1 44 23 39 33
1 8 93 46 17
1 40 78 44 84
1 35 55 23 93
1 19 40 47 8
1 1 42 26 79
1 14 43 26 127
1 30 5 10 88
1 35 127 6 40
1 39 85 41 108
1 49 81 21 49
1 2 123 35 65
1 3 5 2 120
1 47 55 33 94
1 35 32 49 60
1 48 59 30 126
1 1 32 27 102
1 43 15 10 128
1 23 45 24 21
1 5 87 8 94
1 12 104 31 43
1 15 104 26 62
1 40 92 39 36
1 35 31 38 65
1 42 38 29 74
1 2 49 26 47
1 37 21 40 107
1 17 28 5 114
1 42 37 16 118
1 34 119 17 6
1 24 111 5 100
1 15 98 17 14
1 44 75 29 60
1 29 60 8 9
1 22 78 1 43
1 3 57 15 84
1 25 2 40 95
