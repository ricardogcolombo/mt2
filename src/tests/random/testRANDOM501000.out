0.465611
0.536253
0.467735
0.359066
0.488017
0.506985
0.446897
0.553432
0.431068
0.594834
0.660573
0.481967
0.512689
0.523889
0.572524
0.554048
0.495636
0.550829
0.571749
0.607973
0.453213
0.410599
0.483384
0.587269
0.468244
0.604960
0.446771
0.539836
0.684018
0.473595
0.393959
0.605496
0.407407
0.438374
0.483242
0.518328
0.430362
0.406751
0.327805
0.599601
0.417080
0.493005
0.421277
0.591473
0.528993
0.535065
0.405385
0.386712
0.526362
0.549661
